src/main/java/seedu/address/logic/commands/DownloadAllNotesCommand.java

```diff
@@ -13,6 +13,12 @@
 import java.util.List;
 import java.util.NoSuchElementException;
 
+/**
+ * DownloadAllNotesCommand Will download all the notes from a selected module and store it in the "notes" folder.
+ *
+ * DownloadAllNotesCommand extends on DownloadAbstract that extends on command.
+ */
```

Comment:
This is a good description that gives the reader an idea what this class is supposed to do.